File: packages/fixflow_core/lib/models/responses/request_image_response.dart
class RequestImageResponse {
  final int id;
  final String imagePath;
  final String originalFileName;
  final int fileSize;

  RequestImageResponse({
    required this.id,
    required this.imagePath,
    required this.originalFileName,
    required this.fileSize,
  });

  factory RequestImageResponse.fromJson(Map<String, dynamic> json) {
    return RequestImageResponse(
      id: json['id'] as int,
      imagePath: json['imagePath'] as String,
      originalFileName: json['originalFileName'] as String,
      fileSize: json['fileSize'] as int,
    );
  }
}

File: FixFlow/FixFlow.Application/DTOs/Response/RequestImageResponse.cs
namespace FixFlow.Application.DTOs.Response;

public class RequestImageResponse
{
    public int Id { get; set; }
    public string ImagePath { get; set; } = string.Empty;
    public string OriginalFileName { get; set; } = string.Empty;
    public long FileSize { get; set; }
}
